staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -200,6 +200,8 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// runAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
```

Comment:
add `+optional` or existing clients that do client side validation will complain when trying to create objects without this field